src/asynchronous.rs

```diff
@@ -222,7 +222,7 @@ impl AsyncDeviceStateReader {
             StreamDeckInput::ButtonStateChange(buttons) => {
                 for (index, (their, mine)) in zip(buttons.iter(), my_states.buttons.iter()).enumerate() {
                     match self.device.kind {
```

Comment:
I think that this should be an if statement now. The match statement is a lot better for matching 2 or more different conditions but here there's only 1 condition: Mirabox or not.